config/store_params/store_params.config

```diff
@@ -0,0 +1,11 @@
+import groovy.json.JsonOutput
+
+json_extractor {
+      store_params_json = {
+               json_params = JsonOutput.prettyPrint(JsonOutput.toJson(params))
+               File file = new File("${params.log_output_dir}/nextflow-log/params.json")
+               File logdir = new File("${params.log_output_dir}/nextflow-log")
```

Comment:
Can you make this function more modularized to accept any `Map` and file path? The idea being that we'll want to save more than just the params eventually and having separate hard-coded functions for everything wouldn't be ideal

Reply:
Resolved!